app/assets/javascripts/application.js

```diff
@@ -123,6 +123,7 @@ function onContentLoad(){
       if ($(this).data('on-complete')){
         window[$(this).data('on-complete')].call(null, this, status);
       }
+      $(document).trigger('ContentLoad');
```

Comment:
I need multiple hooks (setVersion, data table etc) imho that was the whole
idea of on content load (as new content is loaded) ?